app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,7 @@
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400&display=swap" rel="stylesheet">
     <!-- <link type="text/css" rel="stylesheet" href="../assets/stylesheets/jquery.qtip.css" /> -->
+    <%= javascript_include_tag 'application' %>
```

Comment:
So Josh told me to put this at the bottom so everything that needs to render will do so before hitting the js. It's not 100% required, but I think it's becoming industry standard to put it as the last thing in your body tag.
However, I do question everything now since I couldn't get it to work.